monkey/tests/unit_tests/monkey_island/cc/models/test_agent.py

```diff
@@ -99,6 +103,13 @@ def test_machine_id_immutable():
         a.machine_id = 10
 
 
+def test_registration_time_immutable():
+    a = Agent(**AGENT_SIMPLE_DICT)
+
+    with pytest.raises(TypeError):
+        a.start_time = 100
```

Comment:
This is a `registration_time` test or `start_time`?

Reply:
Nice catch :-)